src/status_im2/common/scroll_page/view.cljs

```diff
@@ -0,0 +1,100 @@
+(ns status-im2.common.scroll-page.view
+  (:require
+   [react-native.core :as rn]
+   [quo2.core :as quo]
+   [utils.re-frame :as rf]
+   [quo2.foundations.colors :as colors]
+   [status-im2.common.scroll-page.style :as style]
+   [oops.core :as oops]
+   [reagent.core :as reagent]
+   [react-native.platform :as platform]))
+
+(defn icon-color []
+  (colors/theme-colors
+   colors/white-opa-40
+   colors/neutral-80-opa-40))
+
+(defn get-platform-value [value] (if platform/ios? (+ value 44) value))
+(def negative-scroll-position-0 (if platform/ios? -44 0))
+(def scroll-position-0 (if platform/ios? 44 0))
+(def scroll-position-1 (if platform/ios? 86 134))
+(def scroll-position-2 (if platform/ios? -26 18))
+
+(defn get-header-size [scroll-height]
+  (if (<= scroll-height scroll-position-2)
+    0
+    (->>
+     (+ (get-platform-value -17) scroll-height)
+     (* (if platform/ios? 3 1))
+     (max 0)
+     (min (if platform/ios? 100 124)))))
+
+(def max-image-size 80)
+(def min-image-size 32)
+
+(defn diff-with-max-min [value maximum minimum]
+  (->>
+   (+ value scroll-position-0)
+   (- maximum)
+   (max minimum)
+   (min maximum)))
+
+(defn icon-top-fn [scroll-height]
+  (if (<= scroll-height negative-scroll-position-0)
+    -40
+    (->> (+ scroll-position-0 scroll-height)
+         (* (if platform/ios? 3 1))
+         (+ -40)
+         (min 8))))
+
+(defn icon-size-fn [scroll-height]
+  (->> (+ scroll-position-0 scroll-height)
+       (* (if platform/ios? 3 1))
+       (- max-image-size)
+       (max  min-image-size)
+       (min max-image-size)))
+
+(defn scroll-page-fn [ios?]
+  (fn  [icon cover page-nav name]
+    (let [scroll-height (reagent/atom negative-scroll-position-0)]
+      (fn [sticky-header children]
+        [:<>
+         [:<>
+          [rn/image
+           {:source   cover
+            :position :absolute
+            :style    (style/image-slider (get-header-size @scroll-height))}]
+          [rn/blur-view (style/blur-slider (get-header-size @scroll-height))]]
+         [rn/view {:style {:z-index 6 :margin-top (if ios? 56 6)}}
```

Comment:
why not use platform/ios? directly here ?

Reply:
its a constant

Reply:
ok, I'll update so. thanks!